operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml

```diff
@@ -9126,7 +9126,7 @@ data:
         readinessProbe:
           httpGet:
             path: /healthz/ready
-            port: {{ annotation .ObjectMeta `status.sidecar.istio.io/port` .Values.global.proxy.statusPort }}
+            port: 15090
```

Comment:
Is this going to break people using annotations?
If so it should go into release notes.

Reply:
I don't think it will. If they set the annotation, to say 1234, the flow will be kubelet -> 15090 -> 1234. They already are using a hardcoded 15090 for stats as well so I am not adding a new hardcoded value